miio/wifirepeater.py

```diff
@@ -86,12 +91,38 @@ def set_wifi_roaming(self, wifi_roaming: bool):
             'wifi_explorer': int(wifi_roaming)
         }])
 
-    def set_configuration(self, ssid: str, password: str, hidden: bool = False,
-                          wifi_roaming: bool = False):
+    def set_configuration(self, wifi_roaming: bool = False, ssid: str = None,
```

Comment:
Would it make sense to have a separate method for enabling and disabling the roaming? Otherwise this looks good to be merged.